test/index.test.js

```diff
@@ -248,9 +248,9 @@ describe('module-resolver', () => {
             'styles/.+\\.(css|less|scss)$': './style-proxy.\\1',
             '^single-backslash': './pas\\\\sed',
             '^non-existing-match': './pas\\42sed',
-            '^regexp-priority': './miss',
+            '^regexp-priority': './hit',
             'regexp-priority$': './miss',
-            'regexp-priority': './hit',
+            'regexp-priority': './miss',
```

Comment:
The test was invalidated because the order is now from the top to the bottom instead of aliases first, then reg exps :)